### DISCUSSION TITLE
Panopticon - Install - Data base Error

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#314] [Q&A (Community Support)]
Trying to install Panopticion - Error on first step !!!
Access denied for user 'o12288402'@'%' to database 'solo' ???
What I don't understand is that neither the username or the database relate to the database I have set up
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/95095238/23019a34-8da7-4017-ae3e-52938834a822)
I entered the Data Base Host Server Name, User name and password. I left the Database Name blank.
I go the error above.
What is the 'Database  Name' I have not had to enter that before? The Data Base is MYSql 8.0
Any ideas??

**Comment by nikosdion** [comment]
You forgot to enter the database name. This is a required field. See the empty field between the password and the database table name prefix.

If the database name is empty it defaults to `solo`. That's because the web framework used in Panopticon (AWF - Akeeba Web Framework) was originally written for Akeeba Solo ten years ago.

**Comment by david-unwin** [comment]
Nicholas....very stupid of me....my apologies for disturbing you.
Kind regards
David

**Reply by nikosdion** [reply]
No worries!
